logbook-json/src/test/java/org/zalando/logbook/json/JsonPathBodyFiltersTest.java

```diff
@@ -0,0 +1,113 @@
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.zalando.logbook.json.JsonPathBodyFilters.jsonPath;
+
+class JsonPathBodyFiltersTest {
+
+    public static final String CONTENT_TYPE = "application/json";
+
+    @Test
+    public void deleteObjectTest() {
+        BodyFilter filter = jsonPath("$.test.test123").delete();
+        String result = filter.filter(CONTENT_TYPE, "{\"test\": {\"test123\":\"testvalue\"}, \"test2\":\"value\"}");
+        assertThat(result).isEqualTo("{\"test\":{},\"test2\":\"value\"}");
+    }
+
+    @Test
+    public void replaceArrayWithStringTest() {
+        BodyFilter filter = jsonPath("$.test").replace("XXX");
+        String result = filter.filter(CONTENT_TYPE, "{\"test\": [\"test123\", \"test321\", 1.0], \"test123\": \"testing\"}");
+        assertThat(result).isEqualTo("{\"test\":\"XXX\",\"test123\":\"testing\"}");
+    }
+
+    @Test
+    public void replaceNumberWithStringTest() {
+        BodyFilter filter = jsonPath("$.test").replace("XXX");
+        String result = filter.filter(CONTENT_TYPE, "{\"test\": 1, \"test123\": \"testing\"}");
+        assertThat(result).isEqualTo("{\"test\":\"XXX\",\"test123\":\"testing\"}");
+    }
+
+    @Test
+    public void replaceArrayWithNumberTest() {
+        BodyFilter filter = jsonPath("$.test").replace(10.0);
+        String result = filter.filter(CONTENT_TYPE, "{\"test\": [\"test123\", \"test321\", 1.0], \"test123\": \"testing\"}");
+        assertThat(result).isEqualTo("{\"test\":10.0,\"test123\":\"testing\"}");
+    }
+
+    @Test
+    public void replaceNumberWithNumberTest() {
+        BodyFilter filter = jsonPath("$.test").replace(10.0);
+        String result = filter.filter(CONTENT_TYPE, "{\"test\": 1, \"test123\": \"testing\"}");
+        assertThat(result).isEqualTo("{\"test\":10.0,\"test123\":\"testing\"}");
+    }
+
+    @Test
+    public void replaceArrayWithBooleanTest() {
+        BodyFilter filter = jsonPath("$.test").replace(true);
+        String result = filter.filter(CONTENT_TYPE, "{\"test\": [\"test123\", \"test321\", 1.0], \"test123\": \"testing\"}");
+        assertThat(result).isEqualTo("{\"test\":true,\"test123\":\"testing\"}");
+    }
+
+    @Test
+    public void replaceNumberWithBooleanTest() {
+        BodyFilter filter = jsonPath("$.test").replace(true);
+        String result = filter.filter(CONTENT_TYPE, "{\"test\": 1, \"test123\": \"testing\"}");
+        assertThat(result).isEqualTo("{\"test\":true,\"test123\":\"testing\"}");
+    }
+
+    @Test
+    public void replaceStringDynamicallyTest() {
+        BodyFilter filter = jsonPath("$.test").replace("(\\d{6})\\d+(\\d{4})", "$1******$2");
+        String result = filter.filter(CONTENT_TYPE, "{\"test\": \"5213486633218931\", \"test123\": \"5213486633218931\"}");
+        assertThat(result).isEqualTo("{\"test\":\"521348******8931\",\"test123\":\"5213486633218931\"}");
+    }
+
+    @Test
+    public void replaceArrayDynamicallyTest() {
```

Comment:
Everything may happen...) 
I always assume that user will do everything wrong :D
I've added new test for wildcard behavior :)

Reply:
>  I always assume that user will do everything wrong :D

You're absolutely right. Given that the feature is intended to prevent sensitive information from leaking into log files, defaulting to *"stringify JSON array/object and replace values"* is a good and safe default. 

We should add docs for the most common use cases.